torch/onnx/_internal/registration.py

```diff
@@ -258,15 +269,13 @@ def all_functions(self) -> Set[str]:
 
 def discover_and_register_all_symbolic_opsets() -> None:
     """Discover all symbolic functions.
-
```

Comment:
Is this expected? I thought there should be blank line.

Reply:
It wasn't. But the function is removed in the next PR anyways. I can fix it if you like

Reply:
I see, nvm unless CI screams.